### ISSUE TITLE
Rain sensor state fails

### ISSUE THREAD

**Issue by trilu2000** [SukramJ/hahomematic#336]
**custom_component/hahomematic version (if applicable):**
0.37.3

**CCU version:**
- Model (CCU2/3, Raspi3/4, NUC, ProxmoxVE, ...):
- OS (Original, RaspberryMatic, debmatic/...):
- Version: 3.61.7.20220226

**Describe the bug**
HM-Sen-RD-O state is not handed over to home assistant correctly, seems the state cannot be converted
[hahomematic.entity] _convert_value: conversion failed 

Init:
```
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] __init__: Initializing device: myCCU-BidCos-RF, OEQ0996025
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] __init__: Initialized device: myCCU-BidCos-RF, OEQ0996025, HM-Sen-RD-O, OG_BAL_RD
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entities: Skipping AES_KEY (no event or internal)
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:0, CONFIG_PENDING, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: binary_sensor: OEQ0996025:0 CONFIG_PENDING
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Ignoring parameter: DEVICE_IN_BOOTLOADER [OEQ0996025:0]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:0, RSSI_DEVICE, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: sensor: OEQ0996025:0 RSSI_DEVICE
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:0, RSSI_PEER, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: sensor: OEQ0996025:0 RSSI_PEER
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:0, STICKY_UNREACH, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: switch: OEQ0996025:0 STICKY_UNREACH
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:0, UNREACH, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: binary_sensor: OEQ0996025:0 UNREACH
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:0, UPDATE_PENDING, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: binary_sensor: OEQ0996025:0 UPDATE_PENDING
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entities: Skipping INSTALL_TEST (no event or internal)
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:1, STATE, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: binary_sensor: OEQ0996025:1 STATE
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.entity] _convert_value: conversion failed for myCCU-BidCos-RF, OEQ0996025:1, STATE, value: [0]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.entity] _convert_value: conversion failed for myCCU-BidCos-RF, OEQ0996025:1, STATE, value: [1]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.entity] _convert_value: conversion failed for myCCU-BidCos-RF, OEQ0996025:1, STATE, value: [0]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Ignoring parameter: INHIBIT [OEQ0996025:2]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entities: Skipping INSTALL_TEST (no event or internal)
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Ignoring parameter: ON_TIME [OEQ0996025:2]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: Creating entity for OEQ0996025:2, STATE, myCCU-BidCos-RF
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entity_and_append_to_device: switch: OEQ0996025:2 STATE
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] create_entities: Skipping WORKING (no event or internal)
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.client] get_cached_paramset: Getting value via JSON-RPC
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.json_rpc_client] _post: Method: Interface.getParamset, [{'interface': 'BidCos-RF', 'address': 'OEQ0996025:0', 'paramsetKey': 'VALUES'}]
2022-03-17 16:38:19 DEBUG (MainThread) [hahomematic.device] init_data: Skipping load_data, missing entities for OEQ0996025.
```

Switch state
```
2022-03-17 16:42:17 DEBUG (Thread-3) [hahomematic.xml_rpc_server] event: interface_id = myCCU-BidCos-RF, channel_address = OEQ0996025:1, parameter = STATE, value = 1
2022-03-17 16:42:17 DEBUG (Thread-3) [hahomematic.entity] _convert_value: conversion failed for myCCU-BidCos-RF, OEQ0996025:1, STATE, value: [1]

2022-03-17 17:04:25 DEBUG (Thread-3) [hahomematic.xml_rpc_server] event: interface_id = myCCU-BidCos-RF, channel_address = OEQ0996025:1, parameter = STATE, value = 0
2022-03-17 17:04:25 DEBUG (Thread-3) [hahomematic.entity] _convert_value: conversion failed for myCCU-BidCos-RF, OEQ0996025:1, STATE, value: [0]
```

**Comment by SukramJ** [comment]
Thanks for reporting.
Should be fixed with 0.37.2

**Comment by trilu2000** [comment]
Thanks for fixing - it is working now
